tools/css-modules-v9-codemod.js

```
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+
+/**
+ * Shakapacker v9 CSS Modules Codemod
+ * 
```

ESLint

[error] Delete `·`

(prettier/prettier)